### DISCUSSION TITLE
感觉wiki里模拟器选择那一节是不是该修整一下了？

### DISCUSSION THREAD

**Discussion by LostMyMind** [LmeSzinc/AzurLaneAutoScript#3033] [General]
这几个月各个模拟器对alas的兼容性又有了比较大的变化，比如：
一、mumu12新版本截图似乎翻车(#2952)，说是后台保活功能的问题，但是该issues的最新评论里好像还是有出现这个问题的人。
二、夜神新版本似乎不好用了(#2584)？后续没关注过，不知道夜神现在修复了没。
三、星界云手机疑似超售导致运行alas不稳。

**Comment by bookbug666** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/Emulator_cn
wiki是可以編輯不用PR的，如果你有資料能證明你不是隨便聽來的，可以考慮打上去幫助後來的人
畢竟wiki也沒有專門的人負責更新 ~~就是別爆破wiki，之前記錄才發現有個SB把條目刪了~~

**Reply by AkiraChisaka** [reply]
对呢，之前特别震撼。

我想安装ALAS的时候，发现中文的安装教程完全消失了……

**Comment by LostMyMind** [comment]
改了星界链云手机的描述，加了新出的那个alas版，还有安卓真机的描述。
其他的主要是我不用这些模拟器，不能确定这些bug修了没，只能等有缘人了。
